lib/global-eval.js

```diff
@@ -60,6 +60,10 @@ var __exec;
       var e;
       window.onerror = function(_e) {
         e = addToError(_e, 'Evaluating ' + load.address);
+        try {
+          onerror.apply(window, arguments);
+        }
+        catch (error) {}
```

Comment:
This is the only change that I've made. The other changes were the result of running `make build`. I wasn't sure the preferred way to handle an error in a custom `window.onerror`. Any preference?